GETTING_STARTED_APP_DEV_CLI.md

```diff
@@ -0,0 +1,205 @@
+
+This section provides application developers wishing to host solutions on Cobalt templates recommendations for building their infrastructure-as-code repository and accompanying CI/CD pipelines. It assumes an isolated Azure DevOps Project with a Cobalt template Repo and Build Pipeline has already been created as defined in the [Getting Started - Advocated Pattern Owner](./GETTING_STARTED_ADD_PAT_OWNER.md) walkthrough.
+
+By creating an application-specific project, you are creating a single project supporting the two main pillars of an application -- the Cobalt-template-based infrastructure and CI/CD build pipeline, and the application code and CI/CD build pipeline. **Important**: as an application developer, you will not be modifying the Cobalt template even though you will be importing all of the required code into your project repository. Instead, you will be responsible only for modifying the configuration via the template's `terraform.tfvars` file to support your application's unique settings (e.g., the number of deployment targets to create or Azure Container Registry image URLs). 
+
+There may be many applications forking from a single governed organization-specific Cobalt template, and requested changes should be made to the source Cobalt template repository so that all applications may update to the latest when needed.
+
+## Prerequisites
+
+* Azure CLI
+  * [Get started with Azure CLI](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli?view=azure-cli-latest)
+* Azure DevOps CLI extension
+  * [Get started with Azure DevOps CLI](https://docs.microsoft.com/en-us/azure/devops/cli/get-started?view=azure-devops)
+* Existing repository and CI/CD pipeline for desired Cobalt template 
+  * [Getting Started - Advocated Pattern Owner](./GETTING_STARTED_ADD_PAT_OWNER.md)
+
+## Steps
+
+```bash
+#
+# 1. Setup Environment Variables
+#
+
+# Define variables shared across commands representing the project to be 
+# created or used, the name of the repository to be created to host Cobalt
+# source in the application project, and the path within the Cobalt source
+# to the build pipeline definition. Make sure to replace the values below
+# in all caps to fit the context of where your organizational Cobalt
+# source template repo is hosted and your application project name.
+export APP_DEVOPS_PROJECT_NAME="APP PROJECT NAME"
+export APP_DEVOPS_INFRA_REPO_NAME="infrastructure"
+export APP_DEVOPS_INFRA_YML_PATH="devops/providers/azure-devops/templates/azure-pipelines.yml"
+export DEFAULT_ORGANIZATION="https://dev.azure.com/ORGANIZATIONNAME/"
+export GIT_SOURCE_URL="https://dev.azure.com/COBALTSOURCEPROJECT/_git/COBALTSOURCEPROJECTREPO"
+
+# These values are used like constants and should not change (unless the
+# build pipeline definition is modified).
+export COBALT_VAR_GROUP_INFRA="Infrastructure Pipeline Variables"
+export COBALT_VAR_GROUP_ENV_SUFFIX="Environment Variables"
+export COBALT_PIPELINE_NAME="Cobalt CICD Pipeline"
```

Comment:
Need to ensure proper string manipulation to handle spaces here. Did not work as written for WSL.

Reply:
Modified usage and tested on bash/zsh on Mac, but still need to test on WSL.